pom.xml

```diff
@@ -525,10 +551,6 @@
       <version>1.21.3</version>
       <scope>test</scope>
       <exclusions>
-        <exclusion>
-          <groupId>org.apache.commons</groupId>
-          <artifactId>commons-compress</artifactId>
-        </exclusion>
```

Comment:
Need to remove this exclusion due to https://issues.jenkins.io/browse/JENKINS-73355.
This dependency is mandatory for testcontainers.

Reply:
`commons-compress`  not included in the HPI:

<img width="508" height="426" alt="Monosnap concordia – ElasticsearchBackendITTest java  opentelemetry  2025-08-07 10-53-45" src="https://github.com/user-attachments/assets/7019fca9-3b36-4e37-9093-3379d5f66ac7" />